theme/static/js/hs_file_browser.js

```diff
@@ -125,6 +126,12 @@ function updateSelectionMenuContext() {
         flagDisableDownload = true;
         flagDisableUnzip = true;
         flagDisableGetLink = true;
+        // Multiple folder deletion is not allowed for composite resource
+        // to avoid race condition that can occur with logical file objects especially
+        // if a folder has many files
+        if(resourceType === 'Composite Resource' && selected.length > 1){
```

Comment:
@mauriel133 It was never possible to select multiple files in composite resource as the metadata for the file can be displayed for only one selected file. So I think this is the desired behavior.